src/Module.cpp

```diff
@@ -546,7 +546,6 @@ void Module::compile(const std::map<OutputFileType, std::string> &output_files)
     if (!submodules().empty() && !should_ignore_submodules(output_files)) {
         std::map<OutputFileType, std::string> output_files_copy = output_files;
         output_files_copy.erase(OutputFileType::stmt);
```

Comment:
Please add a comment explaining why the stmt output is erased (this would have cleared up my confusion).